src/js/polyfills/worker.js

```diff
@@ -9,29 +9,41 @@ class Worker extends EventTarget {
     constructor(path) {
         super();
 
-        const worker = new _Worker(path);
+        const resolve_worker = (resolve, path, blob) => {
```

Comment:
After seeing how this turned out I'm afraid we'd deviate from the standard semantics, for example with regards to terminate.

So, here is another approach, which is somewhat what Deno does:

Use a hidden Symbol to get the parts out of a Blob, as an array. Then you can work with those and create the code string out of them syncrhonously!

Reply:
In addition, not in this PR, we should support data URLs, so what today is called `path` should really be `specifier`, as the standard says.

Reply:
Is what I've done correct? I'm not sure what I'm supposed to do when it comes to hidden symbols. And I thought we could directly create a method for converting text synchronously, instead of accessing the parts and then converting them; to put the conversion work inside the Blob class and in each class that requires it.

Reply:
Not really. You either define the symbol somewhere and import it in both places or you define a global named symbol like so: `Symbol.for('tjs.internal.blob.get-parts')` for example. So if you use it like that in both places, you'll get the same symbol.

Reply:
Ok I will do the changes. However, is the fact that this is a function returning the text ok? Or you prefer the parts? 
PS: Note that what I did works perfectly (I don't know if this is supposed to be the case).

Reply:
Also, when converting to text use TextDecoder, there might be utf-8 inside.

Reply:
I hope, this time, I'm right!